ibm/mas_devops/playbooks/fullstack-roks.yml

```diff
@@ -220,27 +216,22 @@
 
 # 20. Install & Configure MVI application
 # -----------------------------------------------------------------------------
-# MVI support is not yet available for the fullstack configuration as
-# the cluster deployed does not yet configure the required GPU nodes
-#
-# https://github.com/ibm-mas/ansible-devops/issues/48
-#
-# - name: Install Maximo Visual Inspection
-#   import_playbook: mas/install-app.yml
-#   vars:
-#     mas_app_id: visualinspection
-#     mas_app_spec:
-#       settings:
-#         storage:
-#           size: 100Gi
-#           storageClassName': ibmc-file-gold
-# # add to mas_app_spec such as to configure the binding for Edge addon if needed when creating your own playbooks.
-#
-# - name: Configure Maximo Visual Inspection
-#   import_playbook: mas/configure-app.yml
-#   vars:
-#     mas_app_id: visualinspection
-#     mas_workspace_id: masdev
+- name: Install Maximo Visual Inspection
+  import_playbook: mas/install-app.yml
+  vars:
+    mas_app_id: visualinspection
+    mas_app_spec:
+      settings:
+        storage:
+          size: 100Gi
+          storageClassName': ibmc-file-gold
+# add to mas_app_spec such as to configure the binding for Edge addon if needed when creating your own playbooks.
+
+- name: Configure Maximo Visual Inspection
+  import_playbook: mas/configure-app.yml
+  vars:
+    mas_app_id: visualinspection
+    mas_workspace_id: masdev
```

Comment:
Should the workspace id be hardcoded to masdev?

Reply:
I think in the fullstack-roks.yml file it is assumed that the mas_workspace_id is masdev. If the user creates their own playbook, they can change the mas_workspace_id.  I am just using the same workspace id that was created and used by all the other application in this file.

Reply:
oh yes I see what you mean. I didn't notice that in the compare view.